packer/configure-windows-vhd.ps1

```diff
@@ -71,6 +71,7 @@ function Get-FilesToCacheOnVHD
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.8-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.4-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.5-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.7-1int.zip",

```

Comment:
Is this addition Azure Stack-related?

Reply:
yes, I already asked @mboersma to host the file